sonar-batch-protocol/src/main/protobuf/batch_report.proto

```diff
@@ -210,6 +210,24 @@ message Symbols {
   }
 }
 
+message Coverage {
```

Comment:
why do I have the feeling it looks like the graph tests I'm trying to move on compute stack ?